vimrc

```diff
@@ -1,3 +1,6 @@
+" The tests use these settings to ensure consistent results.
+" If you have problems using this plugin, try using these settings.
```

Comment:
Should it be `try changing these settings`? NABD, I can change this myself in case positive. 😉